### ISSUE TITLE
docker-compose.override.yml ignores network_mode

### ISSUE THREAD

**Issue by brikou** [docker/compose#3003]
Here is my configuration:

```yml
# docker-compose.yml
version: '2'

services:
    hello:
        image: alpine
        network_mode: bridge
```

```yml
# docker-compose.override.yml
version: '2'

services:
    hello:
        volumes:
            - .:/mnt
```

Everything is ok with regular docker-compose.yml...

```sh
docker-compose -f docker-compose.yml config
```

```yml
networks: {}
services:
  hello:
    image: alpine
    network_mode: bridge
version: '2.0'
volumes: {}
```

But using docker-compose.override.yml drops network_mode, see...

```sh
docker-compose config
```

```yml
networks: {}
services:
  hello:
    image: alpine
    volumes:
    - /tmp/foo:/mnt:rw
version: '2.0'
volumes: {}
```

I guess this is a bug, anybody can help me to bypass this?

**Comment by brikou** [comment]
Oops I just found #2851, you can close if this is the same issue... Thx

**Comment by dnephin** [comment]
Yup, same issue, fixed in the just released Compose 1.6.1
